StandaloneMmPkg/Drivers/StandaloneMmIplPei/StandaloneMmIplPei.c

```diff
@@ -336,28 +338,35 @@ CreatMmHobList (
   ASSERT (Status == RETURN_BUFFER_TOO_SMALL);
   ASSERT (FoundationHobSize != 0);
 
+  HandOfTableSize = sizeof (EFI_HOB_HANDOFF_INFO_TABLE);
   //
   // Final result includes platform HOBs, foundation HOBs and a END node.
   //

```

Comment:
Update comments to say:
Final result includes: Handoff Info HOB, Platform HOBs, Foundation HOBs and an END node.

Reply:
updated